cap/01-categorie.tex

```diff
@@ -1,9 +1,162 @@
+Nella definizione che segue, a prima vista si può leggere la parola ``classe'' come ``insieme''. Vedremo la questione più in dettaglio nell'Appendice \ref{fondamenti}. 
+
+\begin{definition}
+ Una \emph{categoria} $\ctC$ consiste dei seguenti dati.
+ \begin{itemize}
+  \item Una classe $\ctC_0$ i cui elementi chiamiamo \emph{oggetti}, di solito indicati con lettere latine maiuscole: $A$, $B$, $X$, $Y$,\dots
+  \item Una classe $\ctC_1$ i cui elementi chiamiamo \emph{morfismi}, di solito indicati con lettere latine minuscole: $f$, $g$, $h$,\dots
+  
+  Queste classi sono fornite delle seguenti strutture. 
+  
+  \item Ad ogni morfismo corrispondono due oggetti $\dom{f}$, $\cod{f}$ chiamati \emph{dominio} e \emph{codominio}. Se $f$ ha dominio $X$ e codominio $Y$, scriviamo $f:X\to Y$, o in forma diagrammatica, $X \xrightarrow{f} Y$.
+  \item Ogni oggetto $X$ ha un particolare morfismo $\id_X:X\to X$ chiamato \emph{identità}.
+  \item Dati tre oggetti $X,Y,Z$ e morfismi $f:X\to Y$ e $g:Y\to Z$ (si noti che $\cod{f}=\dom{g}$) esiste un particolare morfismo $g\circ f:X\to Z$ chiamato \emph{composizione di $f$ e $g$}. Graficamente:
+  $$
+  \begin{tikzcd}
+   X \ar{r}{f} \ar[bend right=20]{rr}[swap]{g\circ f} & Y \ar{r}{g} & Z .
+  \end{tikzcd}
+  $$
+  (Si noti che $\dom{g\circ f}=\dom{f}$ e $\cod{g\circ f}=\cod{g}$.)
+  
+  Queste strutture devono soddisfare le seguenti proprietà.
+  
+  \item \emph{Unitalità}: per ogni morfismo $f:X\to Y$, le composizioni $f\circ\id_X$ e $\id_Y\circ f$ sono uguali ad $f$.
+  \item \emph{Associatività}: dati oggetti $X,Y,Z,A$ e morfismi $f:X\to Y$, $g:Y\to Z$ e $h:Z\to A$, le composizioni $h\circ (g\circ f)$ e $(h\circ g)\circ f$ sono uguali.
+ \end{itemize}
+\end{definition}
+
+Dati due oggetti $X$ e $Y$, indichiamo con $\Hom{\ctC}(X,Y)$ la classe di morfismi da $X$ a $Y$. Altri autori usano altre notazioni, come $\mathrm{Hom}(X,Y)$ o $\mathrm{Hom}_\ctC(X,Y)$.
+
+
+Quando le classi $\ctC_0$ e $\ctC_1$ sono insiemi, chiamamo la categoria $\ctC$ \emph{piccola}. Più in generale, $\ctC$ si dice \emph{localmente piccola} se dati ogni due oggetti $X$ e $Y$, la classe $\Hom{\ctC}(X,Y)$ di morfismi da $X$ a $Y$ è un insieme. 
+
+\begin{examples} 
+ Vediamo alcuni esempi classici di categorie. 
+ \begin{itemize}
+  \item Come accennato sopra, gli spazi vettoriali su $\ictR$ e le mappe lineari formano una categoria, che chiamiamo $\ctVect_\ictR$. Più in generale possiamo definire la categoria $\ctVect_\ictF$, per un campo $\ictF$, in maniera analoga.
+  \item Gli insiemi e le funzioni formano una categoria, che chiamiamo $\ctSet$.
+  \item I gruppi e gli omomorfismi di gruppo formano la categoria $\ctGrp$.
+  \item I gruppi abeliani e gli omomorfismi di gruppo formano la sottocategoria $\ctAb\subseteq\ctGrp$. (Definiremo le sottocategorie a breve.)
+  \item Gli spazi topologici e le funzioni continue formano la categoria $\ctTop$. 
```

Comment:
Questo è quello che intendevo sopra: perché non le mappe aperte?

Reply:
Non capisco la domanda. La convenzione è che Top ha come morfismi le funzioni continue. (Anche Set ha le funzioni e non, per esempio, le relazioni...et cetera.) Ma forse non sto capendo il problema... Puoi spiegare meglio?

Reply:
Certo che la convenzione prende le mappe continue! Ma uno che inizia a studiare questo capitolo, forse, potrebbe trarre beneficio dal sentirsi spiegare se a rigor di termini c'è un'altra scelta (le mappe aperte: è vero che fanno di Top_open una categoria?) e veder considerare un esempio dove la scelta dei morfismi non è ovvia dal contesto. In realtà forse mi sto spiegando male di nuovo, è fine giornata e sono stanco: se non è chiaro è colpa mia, edito domani!